api_tests/ledger_query_service/test.js

```diff
@@ -380,7 +380,11 @@ describe("Test Ledger Query Service API", () => {
                     .request(lqs.url())
                     .post("/queries/ethereum/logs")
                     .send({
-                        topics: [[transfer_topic]],
+                        log_matchers: [{
```

Comment:
I guess you are planning on renaming this to "event*" rather than "log*" given your renaming below?

Reply:
that's why this test was failing... :D